core/src/main/scala/com/socrata/thirdparty/scalatra/RequestLogging.scala

```diff
@@ -0,0 +1,15 @@
+package com.socrata.thirdparty.scalatra
+
+import org.scalatra.ScalatraServlet
+import org.slf4j.LoggerFactory
+
+trait RequestLogging extends ScalatraServlet {
+  val logger = LoggerFactory.getLogger(getClass)
+  before() {
+    logger.info(request.getMethod + " - " + request.getRequestURI + " ? " + request.getQueryString)
```

Comment:
We should also consider renaming the serving thread with some of the above info, if that's not already done.